src/test/resources/features/users/delete_users.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Delete Users  
+
+  In order to delete an account
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set valid authentication headers
+
+  @functional
+  @createUser
```

Comment:
Please put tags in a single line, first the corresponding ones for categorization, after the corresponding ones to preconditions or posconditions.
`@functional @createUser`

Reply:
Thanks, It's done.